docker-compose.yaml

```diff
@@ -252,7 +252,7 @@ services:
     image: artifactory.algol60.net/quay.io/coreos/etcd:v3.4.7
```

Comment:
This ETCD instance can be removed once https://github.com/Cray-HPE/csm/pull/1670 is removed